src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -26,34 +26,35 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
         node = await ext.tree.showTreeItemPicker<SubscriptionTreeItem>(SubscriptionTreeItem.contextValue, context);
     }
 
-    await node.runWithTemporaryDescription(
-        context,
-        localize('startingCreate', 'Create Starting...'),
-        async () => {
-            const folder: WorkspaceFolder = await getWorkspaceFolder(context);
-            const gitWorkspaceState: GitWorkspaceState = await getGitWorkspaceState(context, folder.uri);
-            const verifiedWorkspace: VerifiedGitWorkspaceState = await verifyGitWorkspaceForCreation(context, gitWorkspaceState, folder.uri);
-
-            await promptForDefaultBranch(context, verifiedWorkspace.repo);
-            context.telemetry.properties.cancelStep = undefined;
-
-            context.fsPath = folder.uri.fsPath;
-            context.repo = verifiedWorkspace.repo;
-
-            if (gitWorkspaceState.remoteRepo) {
-                context.repoHtmlUrl = gitWorkspaceState.remoteRepo.html_url;
-                context.branchData = { name: gitWorkspaceState.remoteRepo.default_branch };
-            } else {
-                if (!context.advancedCreation) {
-                    // default repo to private for basic create
-                    context.newRepoIsPrivate = true;
-                    // set the org to the authenticated user for creation
-                    context.orgData = await GitHubOrgListStep.getAuthenticatedUser(context);
-                }
+    const progressOptions: ProgressOptions = {
+        location: ProgressLocation.Notification,
+        title: localize('verifyingWorkspace', 'Verifying workspace...')
```

Comment:
I'm pretty sure someone suggested this verbiage instead of "Create Starting" but lemme know if I'm mistaken..

Reply:
Yah that was Mr. Bossman himself, @ejizba